weeb/module.py

```diff
@@ -0,0 +1,145 @@
+        tags = ["Tags", ",".join(str(e) for e in tags)]
+        artists = ["Artists", ",".join(str(e) for e in artists)]
+        groups = ["Groups", ",".join(str(e) for e in groups)]
+        languages = ["Languages", ",".join(str(e) for e in languages)]
+        categories = ["Categories", ",".join(str(e) for e in categories)]
+
+        tags = [
+            characters,
+            parodies,
+            tags,
+            artists,
+            groups,
+            languages,
+            categories
+        ]
+        return tags
+
+    def embed(self, ctx, dic, DOUJIN_ID):
+        tags = self.doujin_tags(dic)
+
+        url = f"https://nhentai.net/g/{DOUJIN_ID}/"
+
+        if dic["images"]["pages"][0]["t"] == "j":
+            cover_url = f"https://i.nhentai.net/galleries/{dic['media_id']}/1.jpg"
+        elif dic["images"]["pages"][0]["t"] == "p":
+            cover_url = f"https://i.nhentai.net/galleries/{dic['media_id']}/1.png"
+
+        title = dic["title"]["pretty"]
+        num_pages = dic["num_pages"]
+
+        embed: discord.Embed = utils.Discord.create_embed(
+            author=ctx.message.author,
+            title=title,
+            url=url,
+            color=discord.Colour.from_rgb(227, 47, 86),
+        )
+        embed.set_image(url=cover_url)
+        embed.add_field(name="Number of pages", value=num_pages, inline=True)
+
+        for typ, tag in tags:
+            if tag != "":
+                if typ == "Tags":
+                    embed.add_field(name=typ, value=tag, inline=False)
+                else:
+                    embed.add_field(name=typ, value=tag, inline=True)
+        return embed
+
+    def is_channel_weeby(self, message):
+        if isinstance(message.channel, PrivateChannel):
+            return True
+        if message.channel.is_nsfw:
+            return True
+        return False
+
+    @commands.cooldown(rate=5, per=20.0, type=commands.BucketType.user)
+    @commands.command()
+    async def sauce(self, ctx):
+        """Give a magic number and you will get your sauce """
+        message = ctx.message
+        args = message.content.split(" ")
+
+        if not self.is_channel_weeby(message):
+            raise commands.MissingPermissions
+        else:
+            if len(args) != 2:
+                await ctx.send(
+                    ">>> " + _(ctx, "Supply me with a magic number and i will give you what you desire")
+                )
+                return
+            DOUJIN_ID = args[1]
+            async with ctx.typing():
+                try:
+                    async with aiohttp.ClientSession() as session:
+                        async with session.get(
+                                f"https://nhentai.net/api/gallery/{DOUJIN_ID}") as response:
+                            if response.status != 200:
+                                return await ctx.reply(_(ctx,"Command encountered an error (E{code}.".format(code=response.status)))
```

Comment:
This line would fail the translation. You have to format the output of the `_()` function.